tests/versioned/apollo-server-hapi/apollo-server-hapi-setup.js

```diff
@@ -14,7 +14,7 @@ const { getTypeDefs, resolvers } = require('../../data-definitions')
 
 const WEB_FRAMEWORK = 'Hapi'
 
-function setupApolloServerHapiTests({suiteName, createTests}, config) {
+function setupApolloServerHapiTests({suiteName, createTests, pluginConfig}, config) {
```

Comment:
Do we need to update the koa and fastify versioned test setup as well?

Reply:
Yeah, those will need to be updated. They didn't exist yet when I made these changes :P

Reply:
I can make those updates with my lambda versioned test PR.

Reply:
i'd be all about not having to do it but those tests will likely break on main if this is merged as the default is to now exclude scalars (except for the top level)